rust/src/transformations/sum/mod.rs

```diff
@@ -79,18 +81,24 @@ where
 // make_(sized_)?bounded_int_(checked|monotonic|ordered|split)_sum
 // make_(sized_)?bounded_float_(checked|ordered)_sum
 
-type BoundedSumTrans<MI, T> =
-    Transformation<VectorDomain<BoundedDomain<T>>, AtomDomain<T>, MI, AbsoluteDistance<T>>;
```

Comment:
These typedefs were pretty handy. The macro rules are kind of long now, and don't get adjusted by rustfmt. Is it worth reintroducing a type `SumTrans<MI, T>`?

Reply:
I didn't like this alias, because it made it so that the python doc comments don't tell you about the types of the supporting elements. The proc-macro can't see through the alias.

Reply:
Also, I'm going to fuse the sized and unsized sums before too long.

Reply:
Ah interesting, OK.